apps/web/app/features/stacking/components/stacking-info-card.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from 'react';
+
+import { HStack, VStack, styled } from 'leather-styles/jsx';
+
+import { Button, CheckmarkCircleIcon, CircleIcon } from '@leather.io/ui';
+
+export interface StackingInfoCardProps {
+  poolAmount: string;
+}
+
+export function StackingInfoCard({ poolAmount }: StackingInfoCardProps) {
```

Comment:
is it really an "info card" if there are interactive steps?

Reply:
StackingStepsCard ?